distribution/docs/src/main/jdocs/content/_security-framework/realms-contents.adoc

```diff
@@ -0,0 +1,107 @@
+
+.Other implementations of the Security API within ${branding}
+[cols="1m,2" options="header"]
+|===
+
+|Classname
+|Description
+
+|org.codice.${ddf-branding-lowercase}.platform.filter.delegate.DelegateServletFilter
+|The `DelegateServletFilter` detects any servlet filters that have been exposed as OSGi services implementing `org.codice.ddf.platform.filter.SecurityFilter` and places them in-order in front of any servlet or web application running on the container.
+
+|org.codice.${ddf-branding-lowercase}.security.filter.websso.WebSSOFilter
+|This filter serves as the main security filter that works in conjunction with a number of handlers to protect a variety of contexts, each using different authentication schemes and policies.
+
+|org.codice.${ddf-branding-lowercase}.security.handler.saml.SAMLAssertionHandler
+|This handler is executed by the WebSSOFilter for any contexts configured to use it.
+This handler should always come first when configured in the Web Context Policy Manager, as it provides a caching capability to web contexts that use it.
+The handler will first check for the existence of an HTTP Authorization header of type SAML, whose value is a Base64 + deflate SAML assertion.
+If that is not found, then the handler will check for the existence of the deprecated `org.codice.websso.saml.token` cookie with the same value.
+Failing that, it will check for a JSESSIONID cookie to use as a reference to a cached assertion.
+If the JSESSIONID is valid, the `SecurityToken` will be retrieved from the cache.
+
+|org.codice.${ddf-branding-lowercase}.security.handler.basic.BasicAuthenticationHandler
+|Checks for basic authentication credentials in the http request header.
+If they exist, they are retrieved and passed to the `LoginFilter` for exchange.
+
+|org.codice.${ddf-branding-lowercase}.security.handler.pki.PKIHandler
+|Handler for PKI based authentication.
+X509 chain will be extracted from the HTTP request and converted to a `BinarySecurityToken`.
+
+|org.codice.${ddf-branding-lowercase}.security.handler.guest.GuestHandler
+|Handler that allows guest user access via a guest user account.
+The guest account credentials are configured via the org.codice.${ddf-branding-lowercase}.security.claims.guest.GuestClaimsHandler.
+The `GuestHandler` also checks for the existence of basic auth credentials or PKI credentials that might be able to override the use of the guest user.
+
+|org.codice.${ddf-branding-lowercase}.security.filter.login.LoginFilter
+|This filter runs immediately after the WebSSOFilter and exchanges any authentication information found in the request with a Subject via Shiro.
+
+|org.codice.${ddf-branding-lowercase}.security.filter.authorization.AuthorizationFilter
+|This filter runs immediately after the `LoginFilter` and checks any permissions assigned to the web context against the attributes of the user via Shiro.
```

Comment:
❓ Who is `Shiro`?

Reply:
I added some information about Shiro to the paragraph above, but I can make this a link too if it is clearer.